tutorials/Atmos/burgers_single_stack.jl

```diff
@@ -4,28 +4,29 @@
 
 # ```math
 # \begin{align}
-# \frac{∂ ρ}{∂ t} =& - ∇ ⋅ (ρu) \\
-# \frac{∂ ρu}{∂ t} =& - ∇ ⋅ (-μ ∇u) - ∇ ⋅ (ρu u') - γ[ (ρu-ρ̄ū) - (ρu-ρ̄ū)⋅ẑ ẑ] \\
-# \frac{∂ ρcT}{∂ t} =& - ∇ ⋅ (-α ∇ρcT) - ∇ ⋅ (u ρcT)
+# \frac{∂ ρ}{∂ t} =& - ∇ ⋅ (ρ\mathbf{u}) \\
+# \frac{∂ ρ\mathbf{u}}{∂ t} =& - ∇ ⋅ (-μ ∇\mathbf{u}) - ∇ ⋅ (ρ\mathbf{u} \mathbf{u}') - γ[ (ρ\mathbf{u}-ρ̄\mathbf{ū}) - (ρ\mathbf{u}-ρ̄\mathbf{ū})⋅ẑ ẑ] \\
+# \frac{∂ ρcT}{∂ t} =& - ∇ ⋅ (-α ∇ρcT) - ∇ ⋅ (\mathbf{u} ρcT)
 # \end{align}
 # ```
 
 # Boundary conditions:
 # ```math
 # \begin{align}
 # z_{\mathrm{min}}: & ρ = 1 \\
-# z_{\mathrm{min}}: & ρu = 0 \\
-# z_{\mathrm{min}}: & T = T_{\mathrm{fixed}}/(ρc) \\
+# z_{\mathrm{min}}: & ρ\mathbf{u} = 0 \\
+# z_{\mathrm{min}}: & ρcT = T_{\mathrm{fixed}}/(ρc) \\
 # z_{\mathrm{max}}: & ρ = 1 \\
-# z_{\mathrm{max}}: & ρu = 0 \\
+# z_{\mathrm{max}}: & ρ\mathbf{u} = 0 \\
```

Comment:
Thanksfor changing this. I think now we have to put the \rho c multiplying T_fixed on the RHS.

Reply:
Yeah, I thought I had done that, updating now..